src/main/java/meditracker/MediTracker.java

```diff
@@ -43,8 +45,14 @@ public MediTracker(List<String> dailyMedicationList) {
      * Runs the MediTracker application.
      * This method displays a welcome message, reads user commands, and processes them until the user exits the
      * application.
+     *
+     * @throws MediTrackerException If an error occurs during the execution of the application.
+     * @throws ArgumentNotFoundException Argument required not found.
+     * @throws NullPointerException When the command does not exist.
+     * @throws NumberFormatException If the argument of type double is not supplied as type double.
```

Comment:
These exceptions are no longer being thrown, please remove them from the javadoc

Reply:
Thx for the review, removed in c7699f5